app/routes/bestuurseenheid/zitting/notulen.js

```diff
@@ -1,18 +1,26 @@
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
 export default class BestuurseenheidZittingNotulenRoute extends Route {
-  breadCrumb = { title: 'Notulen' };
+  @service store;
 
   async model() {
-    const meeting = this.modelFor('bestuurseenheid.zitting');
-    const meetingNotes = await this.store.query('notulen', {
-      "filter[zitting][:id:]": meeting.id,
-      sort: "-publication.created-on",
-      include: "publication"
-    });
-
-    const notulen = meetingNotes.firstObject;
-    return { notulen, zitting: meeting };
+    //Why was it done this way:
+    //const meeting = await this.modelFor('bestuurseenheid.zitting');
+    //const meetingNotes = await this.store.query('notulen', {
+    //  'filter[zitting][:id:]': meeting.id,
+    //  sort: '-publication.created-on',
+    //  include: 'publication',
+    //});
+    //const notulen = meetingNotes.firstObject;
+    //return { notulen, zitting: meeting };
+    //When this looks easier and works the same:  ?
+    const zitting = await this.modelFor('bestuurseenheid.zitting');
+    const notulen = await zitting.notulen;
+    await notulen.publication;
+    const bestuursorgaan = await zitting.bestuursorgaan;
+    await bestuursorgaan.isTijdsspecialisatieVan;
```

Comment:
This is also a bit confusing. The relationships are loaded but nothing is done with it. Is it simply done to not cause async data loading by the template? If so it might make sense to simply fetch the zitting record again with all the necessary includes. An alternative is to add the includes to the parent route that fetched the original zitting record. I prefer to keep it in the route that needs it though.